core/common/src/ContourDisplay.ts

```diff
@@ -0,0 +1,263 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module Symbology
+ */
+
+import { compareNumbers, Id64String } from "@itwin/core-bentley";
+import { ColorDef, ColorDefProps } from "./ColorDef";
+import { LinePixels } from "./LinePixels";
+
+/** Namespace containing types controlling how contour lines are rendered.
+ * @public
+ * @extensions
+ */
+export namespace ContourDisplay {
```

Comment:
Nitpick but this namespace doesn't do anything except make people have to read/type "Contour" a bunch of extra times.